From fa8ae0623e656d94d025d30573dde567ab87a2a3 Mon Sep 17 00:00:00 2001
From: Andy Nichols <andy.nichols@digia.com>
Date: Sun, 23 Sep 2012 14:36:51 +0200
Subject: [PATCH] Fix missing internal module in sync.profile

Build was broken for shadow builds because qtmultimediaquicktools
was not listed in sync.profile

Change-Id: If9d23906dbd37b651a545ae810ee9883b6a23348
---
 sync.profile |    1 +
 1 file changed, 1 insertion(+)

diff --git a/sync.profile b/sync.profile
index 0b2e294..0f5658f 100644
--- a/sync.profile
+++ b/sync.profile
@@ -1,6 +1,7 @@
 %modules = ( # path to module name map
     "QtMultimedia" => "$basedir/src/multimedia",
     "QtMultimediaWidgets" => "$basedir/src/multimediawidgets",
+    "QtMultimediaQuick_p" => "$basedir/src/qtmultimediaquicktools",
 );
 
 %moduleheaders = ( # restrict the module headers to those found in relative path
-- 
1.7.9.5

